lib/python/vdm/tests/server/deployment_test.py

```diff
@@ -87,12 +88,13 @@ def test_get_deployment(self):
         """
         ensure GET Deployment
         """
+        headers = {'Accept': 'application/json'}
```

Comment:
why does the test use json? Or is this additional test. Do you have test for Download with xml?